acra/configuring-maintaining/optimisations/obtaining_metrics.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Obtaining metrics
+weight: 9
+---
+
+# Obtaining metrics
+
+Acra services provide some metrics for Jaeger when configured to do so. This includes
+[AcraConnector]({{< ref "acra/configuring-maintaining/general-configuration/acra_connector/_index.md#jaeger" >}}),
+[AcraServer]({{< ref "acra/configuring-maintaining/general-configuration/acra_server/_index.md#jaeger" >}}),
+[AcraTranslator]({{< ref "acra/configuring-maintaining/general-configuration/acra_translator/_index.md#jaeger" >}}).
+
+# Metrics
+
+<!-- decryptor/postgresql/pg_decryptor.go and decryptor/mysql/response_proxy.go -->
+* `ProxyClientConnection` checks every client request using AcraCensor, if request is allowed, sends it to the database (PostgreSQL or MySQL)
```

Comment:
it's time of processing incomming connection from client to database through AcraServer. This trace starts after accepting TCP connection + SecureSession handshake if used AcraConnector and ends after closing connection from client, database or AcraServer side.